azurerm/internal/services/network/tests/data_source_virtual_network_test.go

```diff
@@ -87,6 +110,30 @@ data "azurerm_virtual_network" "test" {
 `, data.RandomInteger, data.Locations.Primary, data.RandomInteger)
 }
 
+func testAccDataSourceArmVirtualNetwork_basic_addressPrefixes(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctest%d-rg"
```

Comment:
```suggestion
  name     = "acctest%d-n-rg"
```
Updating for best practices